sourmash/lca/command_index.py

```diff
@@ -257,7 +257,7 @@ def index(args):
         sys.exit(1)
 
     # check -- did the signatures we found have any hashes?
-    if not db.hashval_to_idx:
+    if not len(db.hashval_to_idx):
```

Comment:
just to make sure I look into this later - in Python, these two statements (the old and the new) are synonymous, if an object has a length then 'if obj' looks at the length. Why change it here?

Reply:
That was a mistake due to some testing I was doing, will remove!